src/main/java/com/laborete/LaboreteAPI/posts/controller/PostsController.java

```diff
@@ -0,0 +1,55 @@
+package com.laborete.LaboreteAPI.posts.controller;
+
+import com.laborete.LaboreteAPI.posts.models.CreatePostDTO;
+import com.laborete.LaboreteAPI.posts.models.PostDTO;
+import com.laborete.LaboreteAPI.posts.service.PostsService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import java.util.List;
+import java.util.UUID;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping({"v1/posts"})
+@Api(
+        value = "Posts",
+        produces = "application/json",
+        tags = {"Posts"},
+        description = "Posts Controller"
+)
+public class PostsController {
+    @Autowired
```

Comment:
You inejct `PostsService` via constructor bellow. Therefore there is no need for `@Autowired` here. Decide between `@Autowired` and constructor injection.